oaknational/Oak-Web-Application
# Refactor: use OakIcon for subject and resource icons [LESQ-970]

## Description

Music year: {owa_music_year}

Refactor all instances of subject and resource icons in OWA to use the OakIcon from oak-components, this will enable designers to update the icons in cloudinary and have them consistent across the site.

Some of the old pixel sizes in use are not supported in the new components so there are some slight visual changes, lesson overview header is the biggest change.

A few curriculum squad components were updated where they were using SubjectIcon.


## How to test

https://deploy-preview-2863--oak-web-application.netlify.thenational.academy/

You should be able to see icons now coming from cloudinary in the network tab. 
There should be no functionality changes, only small visual changes on the following pages:
- Teachers homepage tab: resource icons on the image
- All subject listing pages (specialist included): subject icons
- Unit listing pages: subject icon in header
- Lesson listing pages: subject icon in header and resource icons on lesson cards
- Lesson overview pages: subject icon in header
- Download and share pages: resource icons
- Curriculum header: subject icon
- Curriculum overview tab: subject icon
- Curriculum download: subject icon (seems to be maths for all subjects)
